apm-agent-core/src/main/java/co/elastic/apm/agent/bci/methodmatching/TraceMethodInstrumentation.java

```diff
@@ -110,6 +111,10 @@ public static void onMethodExit(@Advice.Local("span") @Nullable AbstractSpan<?>
             .and(not(nameContains("CGLIB")))
             .and(not(nameContains("EnhancerBy")))
             .and(not(nameContains("$Proxy")))
+            .and(not(nameStartsWith("java.")))
+            .and(not(nameStartsWith("com.sun.")))
+            .and(not(nameStartsWith("sun.")))
+            .and(not(nameStartsWith("jdk.")))
```

Comment:
I've removed the restriction to instrument core Java classes from the global exclude list as plugins do sometimes need to instrument them. However, I think it still makes sense to have that restriction for `trace_methods`.

Reply:
It could be powerful to have this capability in `trace_methods`. Users can abuse this config regardless. I think the benefit of being able to trace specific methods from these packages exceeds the value of protecting users.

Reply:
alight, removed 🙂

Reply:
😘